n3fit/runcards/examples/nnpdf40-like.yml

```diff
@@ -131,15 +133,16 @@ parameters: # This defines the parameter dictionary that is passed to the Model
 
 fitting:
   fitbasis: EVOL
+  savepseudodata: True
   basis:
-  - {fl: sng, trainable: false, smallx: [1.092, 1.118], largex: [1.479, 3.037]}
-  - {fl: g, trainable: false, smallx: [0.8048, 1.053], largex: [2.828, 5.503]}
-  - {fl: v, trainable: false, smallx: [0.4678, 0.744], largex: [1.547, 3.319]}
-  - {fl: v3, trainable: false, smallx: [0.1268, 0.4549], largex: [1.725, 3.38]}
-  - {fl: v8, trainable: false, smallx: [0.5864, 0.7822], largex: [1.533, 3.282]}
-  - {fl: t3, trainable: false, smallx: [-0.436, 1.0], largex: [1.771, 3.47]}
-  - {fl: t8, trainable: false, smallx: [0.5878, 0.874], largex: [1.548, 3.265]}
-  - {fl: t15, trainable: false, smallx: [1.087, 1.146], largex: [1.493, 3.441]}
+  - {fl: sng, trainable: false, smallx: [1.091, 1.119], largex: [1.471, 3.021]}
+  - {fl: g, trainable: false, smallx: [0.7795, 1.095], largex: [2.742, 5.547]}
+  - {fl: v, trainable: false, smallx: [0.472, 0.7576], largex: [1.571, 3.559]}
+  - {fl: v3, trainable: false, smallx: [0.07483, 0.4501], largex: [1.714, 3.467]}
+  - {fl: v8, trainable: false, smallx: [0.5731, 0.779], largex: [1.555, 3.465]}
+  - {fl: t3, trainable: false, smallx: [-0.5498, 1.0], largex: [1.778, 3.5]}
+  - {fl: t8, trainable: false, smallx: [0.5469, 0.857], largex: [1.555, 3.391]}
+  - {fl: t15, trainable: false, smallx: [1.081, 1.142], largex: [1.491, 3.092]}
```

Comment:
Did you take these from a particular fit?

Reply:
From `NNPDF40_nnlo_as_01180_qcd`